app/studio/[studioID]/page.tsx

```diff
@@ -39,12 +38,10 @@ export default async function StudioPage() {
 
   useEffect(() => {
     fetchStudioData(studioID, setStudioInfo);
-  }, []);
+  }, [studioID]);
```

Comment:
You can actually convert this page file to a server component and use getServerSession instead. This will result in faster load time on the user side. https://next-auth.js.org/configuration/nextjs#unstable_getserversession

You can see my implementation of this in app/user/[userID]/schedule/page.tsx

Reply:
I noticed that you are querying the pages/api, and that's great for a client side component, but in a server component, you would move the querying logic inside here similar to my schedule page.